src/EFCore.PG/Metadata/NpgsqlIndexAnnotations.cs

```diff
@@ -53,5 +54,34 @@ public string Method
         protected virtual bool SetMethod(string value)
             => Annotations.SetAnnotation(NpgsqlAnnotationNames.IndexMethod, value);
 
+        /// <summary>
+        /// The PostgreSQL operators to be used with each index field.
+        /// </summary>
+        /// <remarks>
+        /// https://www.postgresql.org/docs/current/static/indexes-opclass.html
+        /// </remarks>
+        public string[] Operators
+        {
+            get => GetOperators();
+            set => SetOperators(value);
+        }
+
+        protected virtual string[] GetOperators() {
+            var value = (string)Annotations.Metadata[NpgsqlAnnotationNames.IndexOperators];
+            if (value != null) {
+                return value.Split(' ');
```

Comment:
For correctness, it would be good to surround each operator class with quotes and delimit (i.e. for the theoretical case where an operator class contains a space...). Take a look at how the metadata for PostgreSQL extensions is implemented.

Reply:
https://github.com/npgsql/Npgsql.EntityFrameworkCore.PostgreSQL/blob/dev/src/EFCore.PG/Metadata/PostgresExtension.cs#L158 Do you mean thos functions? Should I extract them somewhere to avoid duplication?

Reply:
Yeah... I wouldn't really bother with extracting them to avoid duplication though.

Reply:
i assume delimit happens during set.